.github/workflows/gemfiles-getter.yml

```diff
@@ -0,0 +1,35 @@
+name: Retrieve Gemfiles
+
+on:
+  workflow_call:
+
+jobs:
+  fetch:
+    name: Retrieve Gemfiles
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: Remove existing Gemfile, Gemfile.lock, gemspec and /lib
+      run: rm -rf Gemfile Gemfile.lock relaton-cli.gemspec lib/
```

Comment:
I can understand why `Gemfile.lock` maybe can be removed, but why do we need to remove `gemspec` and `lib` right after the clone?

Reply:
These files are kept up-to-date in another repository (bibxml-service). Here we just download them, commit to the repo and run bundle install. Next time the workflow runs, these files are deleted from this repo and downloaded again from the bibxml-service, this way we can ensure they are always up-to-date.